SYCL/Reduction/reduction_range_1d_dw.cpp

```diff
@@ -1,26 +1,22 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -Xsycl-target-backend=nvptx64-nvidia-cuda --cuda-gpu-arch=sm_60
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
 
-// TODO: accelerator may not suport atomics required by the current
-// implementation. Enable testing when implementation is fixed.
-// RUNx: %ACC_RUN_PLACEHOLDER %t.out
+#include "reduction_utils.hpp"
 
 // This test performs basic checks of parallel_for(range<1>, reduction, func)
-// with reductions initialized with 0-dimensional discard_write accessor
+// with reductions initialized with 1-dimensional discard_write accessor
```

Comment:
I think this comment is not correct for SYCL 2020. reduction is not initialized with an accessor, they may be initialized with a buffer and have a property telling that it is initialized with identity.

Reply:
Please fix the comment in other tests as well.